### ISSUE TITLE
backslashes before markers

### ISSUE THREAD

**Issue by green-rail** [Shougo/neosnippet.vim#355]
I've tried to do it this way:
https://github.com/Shougo/neosnippet.vim/issues/232
but it doesn't seem to work on my system (windows 10, GVIM).
So I have this snippet:
`video_${1:name} video_file="data\scenes\\\${2:file}.ogv"`           //trying to escape the backslash
And when I use it, instead of:
`video_some_name video_file="data\scenes\some_file_name.ogv"`
I got this:
`video_some_name video_file="data\scenes\${2:file}.ogv"`    //the marker is shown as a literal string

So instead of ignoring the second slash, it just got rid of the first one.
Well since there's no issues on this topic, most probably it's just me doing something wrong.
Any advice or comment would be great. Thanks in advance.

**Comment by Shougo** [comment]
Note:
Please use the code block.
It is hard to read.

For this issue, it is feature.

``video_${1:name} video_file="data\scenes\\${2:file}.ogv" is the correct.

**Comment by green-rail** [comment]
Sorry, didn't notice this feature(meaning the code block). And of course it's two slashes, not three in my snippet, so it's actually looks like this:
` <video_${1:name} video_file="data\scenes\\${0:file}/>`
Yes, it's an xml tag.

And it still works as I described: it deletes one slash, and doesn't recognize the marker after it:
`<video_some_name video_file="data\scenes\${0:file}/>`

**Comment by mlessnau** [comment]
I don't think this has been resolved. I just can't get the backslash expansion working in my scenario:

What I want to do:
INPUT: namespace foo::<cursor>
EXPECTED OUTPUT: namespace foo\<cursor>

This is the snippet:
snippet ::
options word
    \${0:TARGET}

It just doesn't matter how many backslashes I'm going to add in order to try escaping the sequence "\$", the marker is always broken and I'm ending up with that:

ACTUAL OUTPUT: namespace foo\${<cursor>0:TARGET}

I really tried all kinds of modifications like different marker styles (${1}, ${1:#:...}, ...). I also tried to replace the plain "\" by a `system('echo \\')` but the result was the same: the backslash before the marker's $ sign broke the snippet expansion.

Maybe it's not an issue with the snippet definition but rather the expansion of it.
